pkg/session/token_store_test.go

```diff
@@ -0,0 +1,183 @@
+package session
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestCleanupOld(t *testing.T) {
```

Comment:
In addition to these, I think it would be nice to have tests for our token store implementation as well. We can create a Kube connector using a fake controller-runtime client. Example here - https://github.com/percona/everest/blob/main/pkg/kubernetes/accounts_test.go#L46